packages/icsi-lib/src/helpers.ts

```
@@ -0,0 +1,244 @@
+  const tokenKey = JSON.stringify(tokenType);
+  const filteredTransactions = allTransactions.filter(
+    (tx) => JSON.stringify(tx.token_type) === tokenKey
+  );
+
+  // Map to a more user-friendly format
+  return filteredTransactions.map((tx) => {
+    let amount = BigInt(0);
+    let from = '';
+    let to = '';
+
+    if (tx.operation && tx.operation[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)